storage/Devices/Partition.h

```diff
@@ -71,9 +71,28 @@ namespace storage
 	unsigned int get_id() const;
 	void set_id(unsigned int id);
 
-	bool get_boot() const;
+	/**
+	 * Query the boot flag of the partition.
+	 */
+	bool is_boot() const;
+
+	/**
+	 * Set the boot flag of the partition. Only supported on Msdos. Note:
+	 * The boot flag known in parted for Gpt sets the partition id to ESP
+	 * and is not a discrete flag.
```

Comment:
What do you mean with "is not a discrete flag"?... In general, I'm not sure if I understand the whole note.